crates/control/src/behavior/look_around.rs

```diff
@@ -1,24 +1,38 @@
-use spl_network_messages::GamePhase;
+use spl_network_messages::{GamePhase, GameState};
 use types::{
-    filtered_game_controller_state::FilteredGameControllerState, motion_command::MotionCommand,
+    filtered_game_controller_state::FilteredGameControllerState,
+    game_controller_state::GameControllerState, motion_command::MotionCommand,
     primary_state::PrimaryState, world_state::WorldState,
 };
 
-pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
-    match (
-        world_state.filtered_game_controller_state,
-        world_state.robot.primary_state,
-    ) {
-        (
-            Some(FilteredGameControllerState {
-                game_phase: GamePhase::PenaltyShootout { .. },
-                ..
+pub fn execute(
+    world_state: &WorldState,
+    game_controller_state: Option<GameControllerState>,
+) -> Option<MotionCommand> {
+    if let Some(game_controller_state) = game_controller_state {
+        match (
+            world_state.filtered_game_controller_state,
+            game_controller_state.game_state,
```

Comment:
Why do you use the `game_controller_state` and not the `filtered_game_controller_state`?

Reply:
Because if we get the signal from seeing the visual referee the filtered_game_controller_state is updated.